browser/app/js/buckets/BucketList.js

```diff
@@ -39,29 +53,48 @@ export class BucketList extends React.Component {
       }
     }
   }
+  loadNextPage() {
+    this.setState({
+      page: this.state.page + 1
+    })
+  }
   render() {
-    const { visibleBuckets } = this.props
+    const { filteredBuckets } = this.props
+    const visibleBuckets = filteredBuckets.slice(0, this.state.page * 100)
     return (
-      <div className="buckets__list">
-        {visibleBuckets.map(bucket => (
-          <BucketContainer key={bucket} bucket={bucket} />
-        ))}
+      <div
+        className="buckets__list"
+        style={{ height: "calc(100vh - 246px)", overflow: "auto" }}
+      >
+        <InfiniteScroll
+          pageStart={0}
+          loadMore={this.loadNextPage.bind(this)}
+          hasMore={filteredBuckets.length > visibleBuckets.length}
+          useWindow={false}
+          element="div"
```

Comment:
It's better to add a className `buckets__scroll` instead of keeping it an independent element.

Reply:
Done